### PR TITLE
Refactor for PATH

### PR DESCRIPTION
**Pull request by profxj**
As titled.  Next PR will be from `associate` to `master`

### CODE REVIEW

File: frb/associate/dev/utils.py [hunk 1]
```diff
@@ -0,0 +1,119 @@
+""" Codes realted to developing what has become PATH (aka astropath)"""
```
Comment (tarraneh) [inline frb/associate/dev/utils.py:1]: realted >> related

File: frb/associate/dev/utils.py [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+import pdb; pdb.set_trace()  # This module is defunct
+
+def add_contam(ncontam, frb_coord, cand_gal, cand_coord, fov,
```
Comment (tarraneh) [inline frb/associate/dev/utils.py:5]: Add comment specifying units for fov, ra, dec

File: frb/associate/dev/utils.py [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+    row['dec'] = c_coord.dec.value
+    cand_gal.add_row(row)
+    # Coordinate
+    cand_coord = SkyCoord([cand_coord, c_coord])
+    # Return
+    return cand_gal, cand_coord
+
+
+
+def mock_run(sky, frbs, sigR, theta, fov, scale_rhalf=2., nsigma=2.,
```
Comment (tarraneh) [inline frb/associate/dev/utils.py:31]: Add comment for args descriptions/units where necessary

**Review by tarraneh — APPROVED** [review]

File: frb/associate/dev/utils.py (hunk 1 above)
Comment (profxj) [reply frb/associate/dev/utils.py:1]: fixed

File: frb/associate/dev/utils.py (hunk 2 above)
Comment (profxj) [reply frb/associate/dev/utils.py:5]: this is basically deprecated code

**Pull request merged**